modules/cnf-intel-n3000-optimize-data-plane.adoc

```diff
@@ -0,0 +1,45 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-understanding-intel-hardware-accelertor-card-for-openshift-container-platform_{context}"]
+= Understanding Intel hardware accelerator cards for OpenShift Container Platform
+
+Hardware accelerator cards from Intel® accelerates 4G/LTE and 5G Virtualized Radio Access Networks (vRAN) workloads, increasing in turn the overall compute capacity of a commercial, off-the-shelf platform.
+
+The hardware accelerator cards detailed in this section are:
+
+* Intel® FPGA PAC N3000
+
+* Intel® vRAN Dedicated Accelerator ACC100
++
+[NOTE]
+====
+The Intel® FPGA PAC N3000 card needs flashed with the 4G/LTE or 5G bitstreams to support vRAN workloads.
+====
+
+.Intel® FPGA PAC N3000
+
+The Intel® FPGA PAC N3000 is a reference FPGA and uses 4G/LTE or 5G Forward Error Correction (FEC) as an example workload accelerating the 5G or 4G/LTE RAN layer 1 (L1) base station network function.
+
+The Intel® FPGA PAC N3000 is a full-duplex, 100 Gbps in-system, re-programmable acceleration card for multi-workload networking application acceleration. It has an optimal memory mixture designed for network functions, with an integrated network interface card (NIC) in a small form factor that enables high throughput, low latency, and low power per bit for a custom networking pipeline.
+
+When the Intel® FPGA PAC N3000 is programmed with a 4G/LTE or 5G bitstream, it exposes the Single Root I/O Virtualization (SR-IOV) virtual function (VF) devices used to accelerate the FEC in the vRAN workload.
+To take advantage of this functionality for a cloud-native deployment, the physical function (PF) of the device must be bound to the `pf-pci-stub` driver to create several VFs. Once the VFs are created, they must also be bound to a DPDK userspace driver (vfio) to allocate them to specific pods running the vRAN workload.
+
+Intel® FPGA PAC N3000 support on {product-title} depends on two Operators:
+
+. OpenNESS Operator for Intel® FPGA PAC N3000 (Programming)
+. OpenNESS Operator for Wireless FEC Accelerators
+
+.vRAN Dedicated Accelerator ACC100
+
+The vRAN Dedicated Accelerator ACC100, based on Intel’s eASIC technology is designed to offload and accelerate the computing-intensive process of forward error correction for 4G/LTE and 5G technology, freeing up processing power.
+Intel eASIC™ devices are structured ASICs, an intermediate technology between FPGAs and standard Application-Specific Integrated Circuits (ASICs).
+
+These devices provide lower unitcost and lower power compared to FPGAs and faster time to market and lower non-recurring engineering cost compared to standard- ASICs.
```

Comment:
s/unitcost/unit cost/

I think you can remove the hyphen after "standard."  FWIW, I doubt that the stated benefits are of interest to the administrative audience for this product documentation.